### PR TITLE
setting core concurrency

### PR DESCRIPTION
**Pull request by joseph-bisonai**
# Description

Waiting Queue need to work together.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Deployment

- [ ] Should publish npm package
- [ ] Should publish Docker image

### CODE REVIEW

File: .github/workflows/core.image+upload.yaml [hunk 1]
```diff
@@ -8,6 +8,9 @@ on:
       - "core/**"
     types:
       - closed  
+  push:
+    branches:
+      - i-525/vrf-redis-setting
```
Comment (martinkersner) [inline .github/workflows/core.image+upload.yaml:13]: Can we fix this branch name?

File: .DS_Store
Comment (martinkersner) [inline .DS_Store:1]: Can you please remove this file?

File: cli/package-lock.json [hunk 2]
```diff
@@ -0,0 +1,10045 @@
+{
```
Comment (martinkersner) [inline cli/package-lock.json:1]: We do not use `npm` in this project, therefore we do not need this lock file.

File: yarn-error.log [hunk 3]
```diff
@@ -0,0 +1,37 @@
+Arguments:
```
Comment (martinkersner) [inline yarn-error.log:1]: Let's not push these files to the repository :)

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (martinkersner) [inline yarn.lock:1]: It looks like this file is not necessary.

**Review by martinkersner — APPROVED** [review]
Looks good! But we will have to cleanup this PR a bit, there are quite many irrelevant files.

File: .DS_Store
Comment (joseph-bisonai) [reply .DS_Store:1]: I removed it~!

**Pull request merged**